primefaces/src/main/resources/org/primefaces/Messages_es.properties

```diff
@@ -1,4 +1,4 @@
-primefaces.calendar.DATE_INVALID_RANGE_MESSAGE_ID = {0}: Error de validación: La fecha de inicio es mayor que la fecha de finalización.
+primefaces.calendar.DATE_INVALID_RANGE_MESSAGE_ID = {0}: Error de validación: La fecha de inicio es posterior a la fecha de fin.
```

Comment:
"Error de validación: La fecha de inicio es posterior a la fecha final.", IMO

Reply:
The same problem again.

If you use the word `inicio`, you should use the word `fin`.
To use the word `final`, it's required to use the word `inicial` and not `inicio`.

Reply:
The right values are

`primefaces.calendar.DATE_INVALID_RANGE_MESSAGE_ID = {0}: Error de validación: La fecha de inicio es posterior a la fecha de fin.`

or

`primefaces.calendar.DATE_INVALID_RANGE_MESSAGE_ID = {0}: Error de validación: La fecha de inicial es posterior a la fecha final.

Reply:
Problem? Show me websites exactly with your translations. I can show you a lot of examples with my suggestions and anyone with yours.
I think should be more neutral like all the projects

Reply:
I work to public administration, I need to follow the RAE rules.

Reply:
@melloware is my last comment here. My final opinion is all should be in a neutral Spanish (like always my suggestions) , if not, we can have 10 translations more, for example Mexican, Bolivian, Chile, etc...

You decide what is the best for the project but if I'm not wrong with primelocale every user can personalise the translations, right?